deployment/terraform/cdn.tf

```diff
@@ -61,6 +61,28 @@ resource "aws_cloudfront_distribution" "cdn" {
     max_ttl                = 300
   }
 
+  ordered_cache_behavior {
+    path_pattern     = "tile/*"
+    allowed_methods  = ["GET", "HEAD", "OPTIONS"]
+    cached_methods   = ["GET", "HEAD", "OPTIONS"]
+    target_origin_id = "originAlb"
+
+    forwarded_values {
+      query_string = true
+      headers      = ["Referer", "X-OAR-Client-Key"]
```

Comment:
I have not found anything to suggest that leaving these headers out of this list will prevent them from getting passed to the origin on cache misses. Do we want different objects to be cached based on referer and client API key? If there isn't anything user specific about the vector tiles, can different users see the same object?

Reply:
Ah, I thought that this was required to get the headers to the origin. We don't need to be invalidating on these. If it is the case that the origin gets all headers on a cache MISS, then I'll set this to `none`.

Reply:
Might be good to test real quick with the `test/` branch just to make sure. I was trying to find something to confirm my expectation that it would still go through, and even though I'm pretty sure it will, I didn't find anything definitive.

Reply:
>Do we want different objects to be cached based on referer and client API key? If there isn't anything user specific about the vector tiles, can different users see the same object?
No to both. Our tiles are not user specific.

These headers are used to make it difficult for code not served from our domain to make unauthenticated API requests. We would like to extend our protection to tiles to prevent other web maps hotlinking to our tile layers.

Reply:
> These headers are used to make it difficult for code not served from our domain to make unauthenticated API requests.

Ah, so in this case we _should_ have changes in those headers invalidate the cache, so that they can be sent to the back-end and be 400'd.

Reply:
Those headers don't need to be involved in the caching process. We just need to make sure that they are always forwarded to the origin.

Reply:
If I'm reading @hectcastro's comments correctly, in case of a cache MISS, all headers are always forwarded to the origin, regardless of this setting. What this setting does is attach these values to the cached version. So a subsequent request with the same value for these two headers will get a cache HIT, but one with different ones will get a MISS and be forwarded to the origin. If we don't specify any headers, then all headers will be ignored during caching. This would allow a certain degree of hotlinking, as while uncached tiles would not be able to be requested, cached tiles could (because Cloudfront would not check the headers before serving the cached tile). To truly make hotlinking difficult, we _should_ include these headers here, so that if their value differs, it causes a fresh request to the origin, which can decide if it should be 200 or 400.